src/averageAges.js

```diff
@@ -14,12 +14,37 @@
  *
  * @return {number}
  */
+
+// helper fn
+function isMan(person) {
```

Comment:
You could declare some of these functions as arrow functions and do implicit returns example:
```
const isMan = (person) => person.sex === 'm'
```
and also destructure the person argument
```
const isMan = ({sex}) =>sex === 'm'
```
But know we can see that `isMan` doesn't actually need the whole person object it could just be
```
const isMan = (sex) => sex === 'm'
```
You just need to pass person.sex to the function call instead of the whole person object

Reply:
You may do something similar for other functions